narwhal/crypto/src/lib.rs

```diff
@@ -43,3 +47,77 @@ pub type NetworkKeyPair = ed25519::Ed25519KeyPair;
 // Type alias selecting the default hash function for the code base.
 pub type DefaultHashFunction = Blake2b256;
 pub const DIGEST_LENGTH: usize = DefaultHashFunction::OUTPUT_SIZE;
+pub const INTENT_MESSAGE_LENGTH: usize = DIGEST_LENGTH + INTENT_PREFIX_LENGTH;
```

Comment:
I'd switch the values to `INTENT_PREFIX_LENGTH + DIGEST_LENGTH;` for consistency with the actual sequence.